pkg/imgutil/imgutil.go

```diff
@@ -238,6 +239,13 @@ func PullImage(ctx context.Context, client *containerd.Client, stdout, stderr io
 				containerd.WithImageHandlerWrapper(source.AppendDefaultLabelsHandlerWrapper(ref, 10*1024*1024)),
 			)
 		}
+		nydus = isNydus(snapshotter)
```

Comment:
I think it is better to refacto this list of snapshotter into switch statement but this can be done in another PR